DER/DerivativesContracts/Options.rdf

```diff
@@ -620,45 +512,46 @@
 	<owl:Class rdf:about="&fibo-fbc-fi-fi;Option">
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasCalculatedMarketValue"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasOptionHolder"/>
+				<owl:onClass rdf:resource="&fibo-der-drc-opt;OptionHolder"/>
 				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;OptionPremium"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasExercisePrice"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasCalculatedMarketValue"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:someValuesFrom rdf:resource="&fibo-der-drc-opt;OptionPremium"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasExerciseSchedule"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;Schedule"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasExercisePrice"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-drc-opt;StrikePrice"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasExerciseStyle"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ex;ExerciseConvention"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasExerciseSchedule"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;Schedule"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fbc-fi-ip;hasLotSize"/>
-				<owl:someValuesFrom rdf:resource="&xsd;decimal"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasExerciseStyle"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ex;ExerciseConvention"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-pas-pas;hasBuyer"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-pas-pas;Buyer"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasOptionWriter"/>
```

Comment:
a bit odd to use in property vs class writer and issuer - why not consistent?

Reply:
@rivettp at the end of our call on Tuesday we made this change.  I am not sure what you want me to change it to, since you agreed to this on Tuesday.  SO - add it to our Friday agenda.

Reply:
My comment was unclear. I meant why use "writer" in the property but "issuer" in the class. We should revisit (not in this PR) in conjunction with what happens if the original issuance is cancelled due to the issuer buying back the option.